index.html

```diff
@@ -0,0 +1,143 @@
+	  			<h1>Beach Front Property - Nebraska</h1>
+	  		</header>
+	  			<section>	
+	  			  	<img src="beach.jpg" alt="">
+	  			  	<p class="P1">Description Text Description Text Description Text Description Text Description Text Description TextDescription Text Description TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription Text
+	  			  	</p>
+	  				<p class="avail">Available</p>
+	  			</section>
+	  		<section class = "sec2">	
+		  		<h2>Specifications</h2>
+		  			<p class="bold">Size: Huge</p>
+		  			<p class="bold">Weight: Weightless</p>
+		  			<footer>Specifications valid until you view property in person</footer>
+		  	</section>
+		  	<section class = "sec3">
+		  		<h3>Pricing</h3>
+		  		<p class="bold">1: Cord of Wood</p>
+		  		<p class="bold">2: A chunk of Frankincense</p>
+	  			<p class="bold">3: Barrels of Wheat</p>
+	  		</section>
+		</article>
+</div>
+<div>
+	<article class="cardbox">
+  		<header>
+  			<h1>Beach Front Property - Iowa</h1>
+  		</header>
+  			<section>	
+  			  	<img src="beach.jpg" alt="">
+  			  	<p class="P1">Description Text Description Text Description Text Description Text Description Text Description TextDescription Text Description TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription Text
+  			  	</p>
+  				<p class="avail">Available</p>
+  			</section>
+	  		<section class = "sec2">	
+		  		<h2>Specifications</h2>
+		  			<p class="bold">Size: Huge</p>
+		  			<p class="bold">Weight: Weightless</p>
+		  			<footer>Specifications valid until you view property in person</footer>
+		  	</section>
+	  	<section class = "sec3">
+		  	<h3>Pricing</h3>
+		  		<p class="bold">1: Cord of Wood</p>
+		  		<p class="bold">2: A chunk of Frankincense</p>
+	  			<p class="bold">3: Barrels of Wheat</p>
+	  	</section>
+	</article>
+</div>
+<div>
+	<article class="cardbox">
+  		<header>
+  			<h1>Beach Front Property - Kansas</h1>
+  		</header>
+  			<section>	
+  			  	<img src="beach.jpg" alt="">
+  			  	<p class="P1">Description Text Description Text Description Text Description Text Description Text Description TextDescription Text Description TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription Text
+  			  	</p>
+  				<p class="avail">Available</p>
+  			</section>
+	  		<section class = "sec2">	
+		  		<h2>Specifications</h2>
+		  			<p class="bold">Size: Huge</p>
+		  			<p class="bold">Weight: Weightless</p>
+		  			<footer>Specifications valid until you view property in person</footer>
+			</section>
+	  		<section class = "sec3">
+		  		<h3>Pricing</h3>
+		  		<p class="bold">1: Cord of Wood</p>
+		  		<p class="bold">2: A chunk of Frankincense</p>
+	  			<p class="bold">3: Barrels of Wheat</p>
+	  		</section>
+	</article>
+</div>
+</div>
+<div class="container2">
```

Comment:
Be careful of naming classes in numerical order like this. If you were to need to add a container between the first and second, you'd end up having to rename the classes.